### PR TITLE
test: add normalizeintensity test

### LINKED ISSUE #44: add NormalizeIntensityTest
add test function for NormalizeIntensity.m operation.

### PR DESCRIPTION
**Pull request by hannatolle**
closes #44

### CODE REVIEW

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Just a minor change in the default test!

File: test/operations/NormalizeIntensityTest.m [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  % specify (arbitrary) input files 
+  pathToWorkspace = '.';
+  inputVolume = 'sample.nii';
+  outputVolume = 'output.nii';
+
+  % these two steps allow us to pass a struct to the operation
+  config = struct('inputVolume', inputVolume, ...
+   'outputVolume', outputVolume, ...
+   'meanIntensity', 1000, ...
```
[the remark this answers is not in the document]
Comment (hannatolle) [reply test/operations/NormalizeIntensityTest.m:19]: hmm yes that's correct. But why are we passing 'verbose' then? I feel like I was passing in default values in many default tests. Now I realize that it doesn't make much sense...

**Review by juancarlosfarah — APPROVED** [review]
👍

File: test/operations/NormalizeIntensityTest.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (juancarlosfarah) [reply test/operations/NormalizeIntensityTest.m]: Indeed. We shouldn't need to pass the values that are defined by default.

**Pull request merged**